tests/text-translator/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
If this PR eventually lands, probably should add the copyright header to even these sort of trivial scripts I guess 🙁